agent/acs/handler/acs_handler.go

```diff
@@ -346,6 +353,20 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 	return client.Serve(acsSession.ctx)
 }
 
+// newHeartbeatTimer creates a new time object, with a callback to
+// disconnect from ACS on inactivity
+func newHeartbeatTimer(client wsclient.ClientServer, timeout time.Duration, jitter time.Duration) ttime.Timer {
```

Comment:
Wondering if this is something we can put into shared wsclient interface, since both ACS and TACS should have a very similar implementation.

Reply:
Yeah, I think I tried that but changed it back. Somehow forgetting what was the reason. One reason I think of is the logs would not be able to indicate correctly which type of client received heartbeat, i.e. TACS or ACS. Let me know if we can work around that, then I can move the handler to wsclient.